### ISSUE TITLE
Add missing breach logos

### ISSUE THREAD

**Issue by obotisan** [mozilla/blurts-server#955]
**Affected versions:**
- Firefox 67.0

**Affected Platforms:**
- Windows 10 x64
- macOS 10.13
- Ubuntu 18.04 x64

**Steps to reproduce:**
1. Navigate to https://fx-breach-alerts.herokuapp.com/
2. Scroll down to the middle of the page. 
3. Look at the "Latest Breach Added" panel. 

**Expected result:**
- The icon for the site is correctly displayed. 

**Actual result:**
- The icon seems to be broken. 

**Additional Notes:**
- Please look at the attached image:
![pic9](https://user-images.githubusercontent.com/42799381/57935426-71d4a900-78ca-11e9-868f-26f531f274e6.png)

**Comment by pdehaan** [comment]
I think these latest logos were just never loaded.

- Missing public/img/logos/Appartoo.png
- Missing public/img/logos/Bukalapak.png
- Missing public/img/logos/ClubPenguinRewritten.png
- Missing public/img/logos/MoreleNet.png

```js
const fs = require("fs");
const path = require("path");

const got = require("got").get;

main();

async function main() {
  const breaches = await getBreaches();
  for (const breach of breaches) {
    const breachLogo = path.join("public", "img", "logos", breach.LogoPath);
    if (!fs.existsSync(breachLogo)) {
      console.error(`Missing ${breachLogo}`);
    }
  }
}

async function getBreaches() {
  const res = await got("https://monitor.firefox.com/hibp/breaches", {json: true});
  return res.body;
}
```

**Comment by GeorgiuCiprian** [comment]
This is verified as fixed on Firefox 67.0 using herokuapp. Tested on across platforms: Win 10 x64, macOS 10.13 and Ubuntu 16.04 x64.
